asf_floorplan_interpreter/pipeline/predict_floorplan.py

```diff
@@ -210,8 +210,7 @@ def predict_labels(
 
         # We found the results are best if the output says at least one kitchen per floorplan
         if correct_kitchen:
-            if "KITCHEN" in label_counts:
-                label_counts["KITCHEN"] = 1
+            label_counts["KITCHEN"] = 1
```

Comment:
I had this previously reflected with the condition at Line 213 included in the API repo [here](https://github.com/nestauk/asf_floorplan_interpreter_api/blob/93b943029dbc4c66d168ad141e00f2cf83ad24a3/app/utils.py#L163-L165)